.github/workflows/unit-tests.yml

```diff
@@ -8,7 +8,13 @@ on:
   pull_request:
     paths:
       - '!docs/**'
-      - '!.github/**'
+      - 'cmd/**'
+      - 'tests/**'
+      - 'pkg/**'
+      - 'internal/**'
```

Comment:
would then skip if we touch the Earthfile?

Reply:
I think that's the minimum requirement - everything we touch _except_ docs should trigger the CI. there are just few cases where we don't want them to trigger

Reply:
That's what I tried to achieve with `!docs/**` according to GH documentation, it should work.
But should is a very rare fish :face_with_spiral_eyes: